templates/docs/examples/patterns/notifications/variants.html

```diff
@@ -11,112 +11,139 @@
+    </p>
   </div>
 </div>
 
 <h4>Borderless inline</h4>
 <div class="p-notification--caution is-borderless is-inline">
   <div class="p-notification__content">
-    <div class="p-notification__title">Title:</div>
-    <div class="p-notification__message">Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.</div>
+    <h4 class="p-notification__title">Title:</h4>
+    <p class="p-notification__message">
+      Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.
+    </p>
   </div>
 </div>
 
 <h4>Collapsed (not implemented)</h4>
 <div class="p-notification--caution is-collapsed">
   <div class="p-notification__content">
-    <div class="p-notification__title">Title</div>
-    <div class="p-notification__message">Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.</div>
+    <h4 class="p-notification__title">Title</h4>
+    <p class="p-notification__message">
+      Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.
+    </p>
   </div>
   <div class="p-notification__meta">
     <div class="p-notification__actions">
-      <button class="p-button--link u-text--muted">View 9+ additional notifications</button>
+      <button class="p-notification__action p-button--link">
+        View 9+ additional notifications
+      </button>
     </div>
   </div>
   <div class="p-notification__stack--caution"></div>
   <div class="p-notification__stack"></div>
 </div>
 
-<h4>Dismissable (not implemented)</h4>
+<h4>Dismissable</h4>
 <div class="p-notification" id="notification">
   <div class="p-notification__content">
-    <div class="p-notification__title">Title</div>
-    <div class="p-notification__message">Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.</div>
-    <button class="p-notification__close" aria-label="Close notification" aria-controls="notification">Close</button>
-  </div>
-  <div class="p-notification__meta">
-    <div class="p-notification__time">1h ago</div>
-    <div class="p-notification__actions">
-      <button>Action</button>
-    </div>
+    <h4 class="p-notification__title">Title</h4>
+    <p class="p-notification__message">
+      Body lorem ipsum dolor sit amet consequiteor. Lorem ipsum dolor sit amet consequiteor.
+    </p>
+    <button
+      aria-controls="notification"
+      aria-label="Close notification"
+      class="p-notification__close"
+    >
+      Close
+    </button>
   </div>
 </div>
 
 <h4>Inline</h4>
 <div class="p-notification--caution is-inline">
   <div class="p-notification__content">
-    <div class="p-notification__title">Title:</div>
-    <div class="p-notification__message">Body copy goes here</div>
+    <h4 class="p-notification__title">Title:</h4>
+    <p class="p-notification__message">Body copy goes here</p>
   </div>
 </div>
 
 <h4>Inline multiline</h4>
 <div class="p-notification--caution is-inline" style="width: 20rem">
```

Comment:
I think we should put more content in the notification, not restrict its size.

Also, we do screenshot examples in Percy in 2 sizes (desktop and mobile), so we are covered when it comes to finding bugs (as long as wrapping happens on 375px width).